### PR TITLE
Configuring SMTP category reorg

### PR DESCRIPTION
**Pull request by ksigler7**
- [x] Taken all valuable information from existing pages and added it to the new category
- [x] Deleted old pages
- [x] Added redirects from deleted page to new content
- [x] Head data should have SEO title, keywords, and description
~- [ ] Screenshots updated~
~- [ ] all images have an alt tag~
~- [ ] all images are named according to style guide~
- [x] Content is in Jobs-to-be-done format
- [x] Update all in-content links and links to deleted pages
- [x] Added additional resources (aim for a link to the glossary, the blog and other jobs-to-be-done)
- [x] Added a TOC
- [x] Added a talk to Dewey/Olivia section
- [x] title optimized for google search snippits
- [x] content optimized for google search snippits
- [x] keyword optimization for each category
- [x] addressed/closed any related issues in the backlog

### CODE REVIEW

File: source/API_Reference/SMTP_API/building_an_smtp_email.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+- Ensure that the header is limited to a maximum total line length of 1,000 characters. Failure to do this can cause intermediate MTA's to split the header on non-space boundaries- this causes inserted spaces in the final e-mail. If your e-mail is going through another MTA before reaching SendGrid, it is likely to have an even lower setting for maximum header length and may truncate the header.
+- When using the API, if our system encounters a parsing error, the message will be bounced to the address specified in the MAIL FROM portion of the SMTP session. The MAIL FROM address is re-written when we send the e-mail out for final delivery, so it is safe to set this to an address that can receive the bounces so that you will be alerted to any errors.
+- When sending Unicode characters via the SMTP API, you should escape these characters using the `\u` escape character. When you do this, Unicode characters like ` á` becomes `\u00E1`. To see a list of Unicode characters in the Wikipedia's [Unicode Article](http://en.wikipedia.org/wiki/List_of_Unicode_characters).
+
+{% anchor h2 %}
+Customizing Your Send (filters)
```
Comment (Whatthefoxsays) [inline source/API_Reference/SMTP_API/building_an_smtp_email.md]: Customizing your send

(capitalization)

File: source/API_Reference/SMTP_API/building_an_smtp_email.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- Ensure that the header is limited to a maximum total line length of 1,000 characters. Failure to do this can cause intermediate MTA's to split the header on non-space boundaries- this causes inserted spaces in the final e-mail. If your e-mail is going through another MTA before reaching SendGrid, it is likely to have an even lower setting for maximum header length and may truncate the header.
+- When using the API, if our system encounters a parsing error, the message will be bounced to the address specified in the MAIL FROM portion of the SMTP session. The MAIL FROM address is re-written when we send the e-mail out for final delivery, so it is safe to set this to an address that can receive the bounces so that you will be alerted to any errors.
+- When sending Unicode characters via the SMTP API, you should escape these characters using the `\u` escape character. When you do this, Unicode characters like ` á` becomes `\u00E1`. To see a list of Unicode characters in the Wikipedia's [Unicode Article](http://en.wikipedia.org/wiki/List_of_Unicode_characters).
```
Comment (Whatthefoxsays) [inline source/API_Reference/SMTP_API/building_an_smtp_email.md]: Fix this guy.

File: source/API_Reference/SMTP_API/building_an_smtp_email.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+Categories allow you to track your emails according to broad topics that you define, like "Weekly Newsletter" or "Confirmation Email". Simply define the category by using the `category` parameter within your X-SMTPAPI header:
+
+{% codeblock lang:json %}
+{
+  "category": "Example Category"
+}
+{% endcodeblock %}
+
+For more information, see our categories documentation.
```
Comment (Whatthefoxsays) [inline source/API_Reference/SMTP_API/building_an_smtp_email.md]: Link?

File: source/API_Reference/SMTP_API/errors_and_troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+- [Response codes](#-Response-codes)
+- [Turning off click tracking](#-Turning-off-click-tracking)
+- [Invalid SMTP API Header](#-Invalid-SMTP-API-Hheader)
+- [Certificate verification failed for smtp.sendgrid.net](#-Certificate-verification-failed-for-smtp.sendgrid.net)
+
+{% anchor h3 %}
+Response codes
+{% endanchor %}
+
+Each SMTP call you make returns a response. `200` responses are usually success responses, `400` are usually deferrals, and we continue to retry `400` messages for up to 72 hours, and `500` responses are hard failures that are not retried by our servers. This table has possible response codes with example errors and a general explanation of that sort of response. This is not an all-inclusive list -  if you are getting an error you don't recognize, check out our [Deliverability center](https://sendgrid.com/deliverabilitycenter/#/).
```
Comment (Whatthefoxsays) [inline source/API_Reference/SMTP_API/errors_and_troubleshooting.md]: Each SMTP call you make returns a response. `200` responses are usually success responses, and `400` responses are usually deferrals. SendGrid continues to retry resending `400` messages for up to 72 hours. `500` responses are hard failures that are not retried by our servers.

**Comment by thinkingserious** [comment]
Hello @ksigler7,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**